dbms/src/Storages/Page/V3/CheckpointFile/CPManifestFileWriter.cpp

```diff
@@ -57,16 +60,27 @@ void CPManifestFileWriter::writeEdits(const universal::PageEntriesEdit & edit)
 void CPManifestFileWriter::writeEditsPart(const universal::PageEntriesEdit & edit, UInt64 start, UInt64 limit)
 {
     const auto & records = edit.getRecords();
+    bool has_data = false;
     CheckpointProto::ManifestFileEditsPart part;
     // In `CPManifestFileReader::readEdits` if `has_more` is false, it will return std::nullopt directly,
     // so we must set `has_more` to be true here and `writeEditsFinish` will write a empty part and set `has_more` to be false.
     part.set_has_more(true);
     for (UInt64 i = 0; i < limit; ++i)
     {
+        // TODO(fap) fix only one entry.
```

Comment:
what is this fix only one entry means?

Reply:
already fixed, it means if the only kind of entry is localkv, we skip the write